### PR TITLE
DCOS 2.2 Release Notes

### PR DESCRIPTION
**Pull request by cmastr**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

https://jira.d2iq.com/browse/D2IQ-71751
 
## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

**Comment by vinodkone** [comment]
@cmastr this is not ready for review right?

**Comment by cmastr** [comment]
> @cmastr this is not ready for review right?

No, still a wip.

File: pages/mesosphere/dcos/2.2/release-notes/2.2/index.md [hunk 1]
```diff
@@ -23,31 +23,10 @@ This release provides new features and enhancements to improve the user experien
...
-
-## Jobs Virtual Networking Support
-Metronome based jobs can now join container networks to communicate with other services/jobs in the same network. For more information, see [Creating Jobs](/mesosphere/dcos/2.2/deploying-jobs/quickstart/).
-
-## Custom Certificates for Admin Router
-DC/OS now allows you to provide a non-CA custom external certificate and key that the Admin Router will then use for clients connecting to a cluster. For more information, see [Configuring a Custom External Certificate](/mesosphere/dcos/2.2/security/ent/tls-ssl/ar-custom/)
-
-## Domain Sockets for Agent Executor Communication
-Added a new configuration option `mesos_http_executors_domain_sockets`, which will cause the mesos-agent to use domain sockets when communicating with executors (default is enabled). This change allows administrators to write firewall rules blocking unauthorized access to the agent port 5051 since access to this will not be required anymore for executors to work.
+## Feature #1
```
[the remark this answers is not in the document]
Comment (greggomann) [reply pages/mesosphere/dcos/2.2/release-notes/2.2/index.md]: @vinodkone did you see the CSI feature description which I committed to this branch previously? I just updated it to include some of your language about data mobility - let me know what you think! I welcome all edits 🙏

[the remark this answers is not in the document]
Comment (vinodkone) [reply pages/mesosphere/dcos/2.2/release-notes/2.2/index.md]: @greggomann no I missed your previous commits. looks good to me. @cmastr lets use Greg's wording.

**Comment by cmastr** [comment]
going to start merging this. If any outstanding comments, open a new PR to address.

**Pull request merged**